Utilities/ReleaseScripts/scripts/git-publish

```diff
@@ -455,7 +455,7 @@ CMSSW_CATEGORIES={"Operations": ["Configuration/Applications",
   "FastSimulation/Configuration", "FastSimulation/EgammaElectronAlgos",
   "FastSimulation/Event", "FastSimulation/EventProducer",
   "FastSimulation/ForwardDetectors", "FastSimulation/HighLevelTrigger",
-  "FastSimulation/L1CaloTriggerProducer", "FastSimulation/MaterialEffects",
+  "FastSimulation/MaterialEffects",
```

Comment:
@smuzaffar Could you confirm this is OK given that `FastSimulation/L1CaloTriggerProducer` package is removed?

Reply:
Yes this is fine